### ISSUE TITLE
Refine prompt language before retraining with expanded training data

### ISSUE THREAD

**Issue by headchem** [headchem/StoryGhostPlotter#42]
Take opportunity to reassess all existing prompts - comparing them against the training data to see if the Advice text makes sense for all examples.

Wherever "flaw" appears in prompts, add in Archetype weaknesses/shadow side language to prompt.

In the training data, do an edit pass to try to infuse more character personality/quirks/flaws into Sequence text. This should make it easier to drop the Character desc/personality/archetype from the prompt in later sequences, because that info will already be infused in the Sequence text.

Only do this after first pass once I've added more training data, and in conjunction with #20 

For Character data, currently, we don't include the log line. If we did, it might result in better fitting characters, given the premise?

Convert prompt generator to Decorator Pattern.

**Comment by headchem** [comment]
prompt ideas:

```Brainstorm some ideas for a dramatic scene between Character1 and Character2```

```
Imagine the following scene. Write the screenplay dialog for this scene. Be creative!

SCENE: scene_desc_here

Character1:
```

The latest "Instruct" is more aligned and boring than the beta or original davinci, so you can get more creative results by using an older engine. If you have a lot of context in the prompt, original davinci can give better results than Instruct.

**Comment by headchem** [comment]
Experiment with the prompt separator ending with "SCRIPT: " or "DESCRIPTION: " instead of current "-> "

Have special prompt layer for Climax with a prompt like "Given the earlier ...SETUP... connect an event in the CLIMAX to an idea foreshadowed in the SETUP" Ask the AI to ret-con in some foreshadowing and ask it to provide a justification for it.

**Comment by headchem** [comment]
For the Log Line description, try something like:

```
This is a story summary about keyword1, keyword2, keyword3. Rewrite the following summary to emphasize that this story is about keyword1, keyword2, keyword3:

[text]

Rewritten summary containing keyword1, keyword2, keyword3: 
```

**Comment by headchem** [comment]
Consider using "chain of thought" language in the prompts, or the sequence data: https://ai.googleblog.com/2022/05/language-models-perform-reasoning-via.html

OLD: "The hero is sad in this scene"
NEW: "Because the previous midpoint was a high point for the hero, things should start to crumble, causing more emotional drama for the reader. Therefore, an event happens that saddens the hero, reversing the false high they felt at the midpoint."

Basically, add more reasoning and justification for the current events. The model should output events that better back up that reasoning as a result.
